### DISCUSSION TITLE
Arkwrights Pub Chat - Sat 2/12/23

### DISCUSSION THREAD

**Discussion by richardbirkin** [SuperHyperInstantFutureTime/TrackShift#136] [Work log]
It's been a while since we met away from screens, so to the pub we went. 

The main point of discussion (that related to TrackShift) was an idea that I've been turning around in my brain for a week or so that relates to users paying to use the product. 

When I think about tiers, I do the natural thing of making a matrix of features that each level of user would want in terms of features. But then I think about the amount of dev that it would take to lock features down to different tiers. It doesn't feel like a very us thing to do. 

Instead, a user can use the product for free without logging in. 

If they want to save their work, they need to Register. This is where we'll put the paywall, because if a user wants to save their work and use TrackShift in the future, then the product is adding value to the user. 

We also advertise an Enterprise offer where we'll work with a larger client to build bespoke features especially for their account. 

So:

Use TrackShift for free, forever (? TBD), but your data is at risk as it is tied to a cookie in your browser. 

Use TrackShift for a small fee and get a secure login and support.

**Comment by g105b** [comment]
Bang on. It makes so much sense and is completely _us_. No arbitrary rules, just pay for the product when you've proven its value.
